models/comments.model.js

```diff
@@ -22,3 +22,16 @@ exports.createComment = (articleID, username, body) => {
   return rows[0];
  });
 };
+
+exports.removeComment = (comment_id) => {
+ const query = `
+   DELETE FROM comments
+   WHERE comment_id = $1
+   RETURNING *;`;
+ return db.query(query, [comment_id]).then(({ rows }) => {
+  if (rows.length === 0) {
+   return Promise.reject({ status: 404, msg: "comment not found" });
+  }
+  return rows[0];
```

Comment:
Is there any reason to return the deleted comment? You don't do anything with it in your controller